docs/reference/mapping/runtime.asciidoc

```diff
@@ -91,7 +91,7 @@ calculates the day of the week based on the value of `timestamp`, and uses
 
 [source,console]
 ----
-PUT my-index/
+PUT my-index-000001/
```

Comment:
We use `my-index` and stuff like that all the time so I don't think its a huge deal. I'm not sure how realistic the `-0000001` suffix is for queries. I think if folks have time based data they'll likely query an alias.